ui/apps/platform/src/Containers/AccessControl/AuthProviders/RuleGroups.tsx

```diff
@@ -57,6 +57,10 @@ function RuleGroups({
 }: RuleGroupsProps): ReactElement {
     const augmentedRuleKeys = getAugmentedRuleKeys(ruleAttributes, groups);
 
+    function isDisabled(group: Group) {
+        return disabled || group?.props?.traits?.mutabilityMode === 'ALLOW_FORCED';
```

Comment:
Same notion as from @misberner's [comment within the previous PR](https://github.com/stackrox/stackrox/pull/2610#discussion_r942385111).